tests/int/test_headers.py

```diff
@@ -0,0 +1,174 @@
+import pytest
```

Comment:
Have you reviewed the policies to confirm what the test is / is not covering?

We discussed in the TA Review that it wasn't covering the the policy that puts the x-request-id in the response. Please add a note to https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-58096 with any coverage that needs adding via an end-to-end test.

Reply:
I've already added a note on the jira.

Reply:
As for the end to end tests I have highlighted the response policies that are not covered which would be end to end tests.

Reply:
Ah I've not added a note to the end to end tests jira just my own. I'll add one to that too.

Reply:
Thanks